tutorials/rise_timeseries_frb.ipynb

```diff
@@ -0,0 +1,506 @@
+{
```

Comment:
<p>Would it be useful to add 1-2 sentences explaining what is seen in the data?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Good idea, I've added an explanation</p>